tests/nested/core20/basic/task.yaml

```diff
@@ -27,4 +30,4 @@ execute: |
     execute_remote "sudo snap remove test-snapd-sh" || true
 
     echo "Ensure 'snap list' works and test-snapd-sh snap is removed"
-    execute_remote "snap list" | NOMATCH test-snapd-sh
+    execute_remote "snap list" | not MATCH test-snapd-sh
```

Comment:
Woah, how did this ever work :?

Reply:
spread master branch has NOMATCH, perhaps someone tried it with one of the recent revisions?

Reply:
Aha, nice. good to know

Reply:
yes, while spread master has NOMATCH I would rather just not use it since it's not available "everywhere"